src/foam/core/reflow/Console.js

```diff
@@ -300,34 +300,40 @@ foam.CLASS({
       padding: 10px 8px;
       align-items: center;
       cursor: pointer;
-      border: 1px solid $grey200;
+      border: 1px solid $borderLight;
       border-radius: 4px;
     }
-    ^ table td .close {
-      font-size: 1.2rem;
-    }
+
     .foam-u2-ActionView-text:hover:not(:disabled) {
-      background-color: $grey400!important;
+      background-color: $backgroundInverseTertiary!important;
```

Comment:
Can i ask why you need this? This is global CSS for all Actions, this will break every text action in the app. If you just want a grey button you can use buttonStyle: 'TERTIARY'